swagger/swagger.js

```diff
@@ -0,0 +1,25 @@
+const dotenv = require('dotenv');
+const swaggerAutogen = require('swagger-autogen')();
+
+dotenv.config();
+
+const doc = {
+  info: {
+    title: 'Wildlife Observation API',
+    description: 'API for wildlife observation',
+  },
+  host: process.env.HOST,
+  produces: ['application/json'],
+  schemes: [process.env.MAIN_SCHEME],
+};
+
+const outputFile = './swagger-output.json';
+
+const routes = [
+  '../routes/animal_r.js',
+  // '../routes/observation_r.js',
+  // '../routes/report_r.js',
```

Comment:
We'll keep these two routes commented out until we implement the corresponding resources. Otherwise, Swagger errors out.